cost-analyzer/templates/cost-analyzer-network-costs-template.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.networkCosts -}}
+{{- if .Values.networkCosts.enabled -}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "cost-analyzer.networkCostsName" . }}
+  labels:
```

Comment:
This block is probably unnecessary given the block on L17-18

Reply:
Ok, so which is recommended? Use template to provide name, labels, or metadata at the root of the yaml?